cloudigrade/account/models.py

```diff
@@ -53,6 +53,28 @@ def product_identifier(self):
         """
 
 
+class MachineImage(BaseModel):
+    """Base Class for A cloud VM image."""
+
+    TYPE = model_utils.Choices(
+        'Windows',
+        'Linux',
+        'Unix-like'
```

Comment:
Amazon distinguishes between a couple types of AMI for billing reasons - 

1) Windows Server (Various flavors are priced differently based on feature set)
2) Named Linux Distros with paid support (e.g. RHEL, SuSE, Ubuntu)
3) "Generic Linux/UNIX" 

I'm not sure if this is relevant to what Cloudigrade is doing, but I'm skeptical that "Unix-like" is going to be a useful part of the model or that a single "Linux" flag is not going to need to be differentiated at some point.

Reply:
A little background: this is really serving as a flag for Windows instances so that we don't go through the trouble of trying to inspect a Windows machine. 

I was actually expecting some input/changes here. I don't like what I have here, but figured let's crowdsource! It could just be a boolean flag for determining if it is Windows. Doesn't actually need string choices. 
  
What is *not* in the model at this time, but will be in a story or two is more information coming back from our image inspection process. I'm leaving the model expansion to that story. At that time we'll likely store results that indicate actual distribution.